homeworks/3/demo/src/main/java/com/example/demo/Login.java

```diff
@@ -0,0 +1,73 @@
+package com.example.demo;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.sql.*;
+
+@WebServlet(name = "login", value = "/login")
+public class Login extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+
+    }

```

Comment:
I recommend you do not override this method since now when the user calls the GET method it won't do anything, and they will wonder what is going on. If the method is not present in the code, the user will get an error message and know that this operation is not valid.